osu.Game/OsuGameBase.cs

```diff
@@ -68,8 +68,7 @@ private void load()
             Dependencies.Cache(API = new APIAccess
             {
                 Username = Config.Get<string>(OsuConfig.Username),
-                Password = Config.Get<string>(OsuConfig.Password),
-                Token = Config.Get<string>(OsuConfig.Token)
+                Token = string.IsNullOrEmpty(Config.Get<string>(OsuConfig.Username)) ? string.Empty : Config.Get<string>(OsuConfig.Token)

```

Comment:
Would be nice to simplify this long line.
What about just `Token = Config.Get<string>(OsuConfig.Token)`? As far as I understand, it will give us the same result.